docs/reference/commandline/load.md

```diff
@@ -23,7 +23,6 @@ Load an image from a tar archive or STDIN
 Options:
       --help           Print usage
   -i, --input string   Read from tar archive file, instead of STDIN.
-                       The tarball may be compressed with gzip, bzip, or xz
```

Comment:
Is there a reason you want to remove these, or just because they are not printed on the command-line?

I think the reason they are in the documentation, but not printed on the command line is that the CLI currently doesn't play nice with multi-line descriptions (something we'll be working on).

I'd rather not remove this information from the documentation if the information is correct